### ISSUE TITLE
This is not a bug report

### ISSUE THREAD

**Issue by marloscaida** [hotbso/openSAM#1]
Hotbso:
This is not a bug report. Just an observation. In some airports developed using the SAM plugin (in which before, some worked and others did not with that plugin, probably due to some programming defect), in most of them OpenSAM does not work for me, but it does work for me in at least one of them. they. So I rule out installation or misuse problems.
What's more, it works very well with Javelin's SGES plugin.
If you want, I can give you two scenarios that do not work in OpenSAM, but one worked in SAM and the other never worked in SAM.
I can also pass you a scenario that works in SAM and with OpenSAM, very well.
Basically, I put myself at your entire disposal to collaborate with its development.
The OpenSam and SGES versions are the latest.

**Comment by hotbso** [comment]
I suggest that for a situation where SAM worked and openSAM not you proceed as described here: [https://forums.x-plane.org/index.php?/files/file/90865-opensam-an-open-source-replacement-for-sam/](url)
